pulsar-broker/src/test/java/org/apache/pulsar/broker/admin/AdminTest.java

```diff
@@ -775,7 +773,7 @@ public void resourceQuotas() throws Exception {
         TenantInfoImpl admin = TenantInfoImpl.builder()
                 .allowedClusters(Collections.singleton(cluster))
                 .build();
-        ClusterDataImpl clusterData = ClusterDataImpl.builder().serviceUrl(cluster).build();
+        ClusterDataImpl clusterData = ClusterDataImpl.builder().serviceUrl("http://example.pulsar").build();
```

Comment:
Is this a breaking change? I haven't reviewed the details.

Reply:
It's not a breaking change; we just didn't have data validation.